Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -288,7 +289,7 @@ internal static void RegisterSystemHandlers(bool hostMode)
         /// <typeparam name="T">The message type to unregister.</typeparam>
         /// <param name="handler"></param>
         /// <param name="requireAuthentication">true if the message requires an authenticated connection</param>
-        public static void RegisterHandler<T>(Action<NetworkConnectionToServer, T> handler, bool requireAuthentication = true) where T : IMessageBase, new()
+        public void RegisterHandler<T>(Action<NetworkConnectionToServer, T> handler, bool requireAuthentication = true) where T : IMessageBase, new()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)